packages/create-ripple-app/generators/updates/1.32.0/saofile.js

```diff
@@ -0,0 +1,92 @@
+const common = require('./../../common')
+const templateDir = './../../../template'
+const jscodeshift = require('jscodeshift')
+
+module.exports = {
+  ...common,
+  templateDir,
+  actions () {
+    const actions = [
+      {
+        type: 'modify',
+        files: 'package.json',
+        handler (data) {
+          data.dependencies['nuxt'] = `2.14.12`
+          data.devDependencies['@babel/eslint-parser'] = `^7.12.1`
+          data.devDependencies['@babel/eslint-plugin'] = `^7.12.1`
+          data.devDependencies['cross-env'] = `^7.0.3`
+          data.devDependencies['eslint'] = `^7.19.0`
+          data.devDependencies['eslint-config-standard'] = `^16.0.2`
+          data.devDependencies['eslint-friendly-formatter'] = `^4.0.1`
+          data.devDependencies['eslint-loader'] = `^4.0.2`
+          data.devDependencies['eslint-plugin-jest'] = `^24.1.3`
+          data.devDependencies['eslint-plugin-node'] = `^11.1.0`
+          data.devDependencies['eslint-plugin-promise'] = `^4.2.1`
+          data.devDependencies['eslint-plugin-standard'] = `^5.0.0`
+          data.devDependencies['eslint-plugin-vue'] = `^7.5.0`
+          // Delete babel-eslint as this is now replaced with @babel/eslint-parser
+          delete data.devDependencies['babel-eslint']
+          return data
+        }
+      },
+      {
+        type: 'modify',
+        files: '.eslintrc.js',
+        handler (data) {
+          const j = jscodeshift
+
+          return j(data).find(j.Property).forEach(prop => {
+            // Update eslint parser from babel-eslint to @babel/eslint-parser
+            if (prop.value.value.value === 'babel-eslint') {
+              prop.value.value.value = '@babel/eslint-parser'
+            }
+
+            // Disable babel configFile
+            if (prop.value.key.name === 'parserOptions') {
+              prop.value.value.properties.push(j.property(
+                'init',
+                j.identifier('requireConfigFile'),
+                j.literal(false)
+              ))
```

Comment:
@tim-yao this should add `requireConfigFile: false` so it won't look for babel-config.